src/prog.js

```diff
@@ -25,7 +25,11 @@ export default handler => {
 		.option('--format, -f', 'Only build specified formats', DEFAULT_FORMATS)
 		.option('--watch, -w', 'Rebuilds on any change', false)
 		.option('--target', 'Specify your target environment (node or web)', 'web')
-		.option('--external', `Specify external dependencies, or 'none'`)
+		.option(
+			'--external',
+			`comma-separated list of regex and module-names, or 'none'`,
```

Comment:
[nit] can we reverse these so that "module names" comes first (because it's simpler/more common)?

```suggestion
			`List of module names or regular expressions to leave as external dependencies, or 'none' to inline everything.`,
```